public_html/wp-content/themes/pattern-directory/package.json

```diff
@@ -54,6 +53,13 @@
 	"prettier": "../../../../.prettierrc.js",
 	"stylelint": {
 		"extends": "../../../../.stylelintrc",
-		"ignoreFiles": ["**/*.css", "**/*.css.map"]
+		"ignoreFiles": [
+			"**/*.css",
+			"**/*.css.map"
+		]
+	},
+	"dependencies": {
+		"@wordpress/data": "^4.27.0",
+		"@wordpress/viewport": "^2.26.0"
```

Comment:
Can you put these in devDependencies with the other wp deps?

Reply:
Ah good catch. Thanks.